packages/uniforms/__suites__/render.tsx

```diff
@@ -11,7 +11,9 @@ export function render(
   schema: SimpleSchemaDefinition,
   contextValueExtension?: Partial<Context<unknown>>,
   initialModel?: object,
-) {
+): RenderResult & {
+  rerenderWithProps: (props: Record<any, any>) => void;
+} {
```

Comment:
we need provide this type because when we don't we have this error

```
TS2742: The inferred type of 'render' cannot be named without a reference to '@testing-library/dom/node_modules/pretty-format/build/types'. This is likely not portable. A type annotation is necessary.
```